### DISCUSSION TITLE
Forward posts?

### DISCUSSION THREAD

**Discussion by leonadams87** [tomer8007/kik-bot-api-unofficial#193] [Ideas]
Is there are a way to forward posts to other groups with this API? I tried using **send_chat_image**, but of course that takes in a file path and not the URL of a post. I know there is **send_link**, but then it will not be in-lined. If it is currently not covered, I hope you will consider adding it! If it is a relative simple extension perhaps you could point me in the right direction and I might try to dive in myself.

**Comment by bluemods** [comment]
Not really. When a content message is forwarded in the app, it copies the existing content metadata from storage and translates it to XMPP to be re-sent in the `content` tag (except it converts camera to gallery for obvious reasons), but there's no specific forward API call.
